### DISCUSSION TITLE
ItemsSource is (null) in the devtools debug window even though it has a Binding

### DISCUSSION THREAD

**Discussion by ozan1318** [AvaloniaUI/Avalonia#12199] [Q&A]
Pretty sure this is just me missing something obvoius but haven't been able to find a solution so far. I have a tab with a ListBox, which has a Binding to an ItemsSource of type ObservableCollection<WindowInfo>, where WindowInfo is an object which holds a string. The axaml is as follows:
```
<ListBox Name="WindowListBox" ... ItemsSource="{Binding windowIdList, Mode=TwoWay}" >
    <ListBox>ItemTemplate>
        <DataTemplate>
            <TextBlock Text="{Binding Path=windowId, Mode=TwoWay}" />
        </DataTemplate>
    </ListBox.ItemTemplate>
</ListBox>
```
The MainViewModel constructor includes: ` windowIdList = newObservableCollection<WindowInfo>();`

And MainWindow.axaml.cs includes: `DataContext = new MainViewModel();`

windowIdList starts empty but is filled later on.

When I use the DevTools debugger and look at the ListBox, the ItemsSource property has a value of (null), even after items are added to windowIdList.

What am I missing here? FYI I have no previous experience with WPF, UWP, WinForms or AvaloniaUI.

**Comment by jp2masa** [comment]
Is `windowIdList` a field? It has to be a property.
